src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package Duke;
+
+import Duke.Task.TaskManager;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void main(String[] args) {
+
+        DataDetective.loadData();
+        printMessage message = new printMessage();
+        message.printIntro();
+        Scanner myObj = new Scanner(System.in);
+        boolean condition = true;
+
+        //The main loop where all the input cases are run through, if there was no specific command given, the statement is added as a task to the list.
+        while (condition) {
+            String userInput = myObj.nextLine();
+            String instruction = userInput.contains(" ") ? userInput.substring(0, userInput.indexOf(" ")): userInput;
+            String instructionTask = userInput.contains(" ") ? userInput.substring(userInput.indexOf(" ") + 1) : " ";
+
+            switch (instruction) {
+            case "bye": // When the command bye is given, the program exits the while loop and stops
```

Comment:
Since the logic seems self-explanatory, avoid using comments to describe the code.